src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+
+  .logo {
+    height: 40px;
+    width: 40px;
+    object-fit: contain;
+    padding: 10px 0;
```

Comment:
The padding in the `.logo` block should also follow the multiple of 4 rule. Adjust the vertical padding to be a multiple of 4.